src/Strings.ts

```diff
@@ -205,6 +218,7 @@ const localizedStrings: LocalizedStrings = {
         send: "傳送",
         unknownFile: "[類型係'%1'嘅文件]",
         unknownCard: "[唔知'%1'係咩卡片]",
+		receiptVat: "Vat",
```

Comment:
It's been a long time since I lived in a place where they charged VAT. Is it typically capitalized as "Vat"?